debian/rules

```diff
@@ -25,6 +25,9 @@ override_dh_auto_install:
 	install -d debian/qubes-core-admin-client/usr/share/man/man1
 	install -m 0644 build/man/* \
 		debian/qubes-core-admin-client/usr/share/man/man1/
+	install -d $(DESTDIR)/etc/qubes/post-install.d
+	install -m 0755 scripts/30-keyboard-layout-service.sh \
+		$(DESTDIR)/etc/qubes/post-install.d/30-keyboard-layout-service.sh
```

Comment:
See other rules (and also travis log), this should place files in `debian/qubes-core-admin-client`, not `$(DESTDIR)`. Or set `DESTDIR` at the beginning of this file, like it's done in other packages.